tests/storage/test_hotplug.py

```
@@ -211,45 +270,66 @@ def test_hotplug_volume_with_bus_and_persist_migrate(
+@pytest.mark.usefixtures("hotplugged_dvs_scope_class")
 class TestHotPlugWithSerialPersist:
-    @pytest.mark.sno
     @pytest.mark.polarion("CNV-6425")
-    @pytest.mark.dependency(name="test_hotplug_volume_with_persist")
-    @pytest.mark.s390x
+    @pytest.mark.dependency(name="test_hotplug_volume_with_serial_and_persist")
     def test_hotplug_volume_with_serial_and_persist(
         self,
-        blank_disk_dv_multi_storage_scope_class,
-        fedora_vm_for_hotplug_scope_class,
+        hotplugged_dvs_scope_class: list[DataVolume],
```

Ruff (0.15.21)

[warning] Unused method argument: `hotplugged_dvs_scope_class`

(ARG002)